keyboards/converter/palm_usb/config.h

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/* This code makes use of cy384's Arduino USB HID adapter for the Palm Portable
+ Keyboard, released under the BSD licence */
+
+
+
+
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#define CUSTOM_MATRIX 2
+
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0001
+#define DEVICE_VER      0x0100
+#define MANUFACTURER    QMK
+#define PRODUCT         Stowaway converter
+#define DESCRIPTION     USB converter for Stowaway keyboard
+
+// IO pins to serial
+// https://deskthority.net/wiki/Arduino_Pro_Micro for pin lookup
+#define VCC_PIN D1 // pro micro 2 
+#define RX_PIN   D0 //pro micro 3 , was 8 on cy384
+#define RTS_PIN  C6 // 5  //[ was D4 // 4 on the cy384
+#define DCD_PIN  E6  //7 
+
+#ifdef CY384
+    #define GND_PIN  D7 //6
+    #define PULLDOWN_PIN  B1 // 15
+#endif
+// Set to 1 for Handspring / to disable RTS/DCD based handshake. 
+#define HANDSPRING 0
```

Comment:
It may be better to do something like this: 
```suggestion
#ifndef HANDSPRING 
  #define HANDSPRING 0
#endif
```

This way, it can be set in other keymaps, but doesn't have to be undefined.